conda_smithy/templates/drone.yml.tmpl

```diff
@@ -0,0 +1,27 @@
+{% if configs[0] -%}
+{%- for config, platform, upload in configs | sort -%}
+---
+kind: pipeline
+name: {{ config }}
+
+platform:
+  os: linux
+  arch: arm64
+
+steps:
+- name: Install and build
+  image: {{ docker.image }}
+  environment:
+    CONFIG: {{ config }}
+    UPLOAD_PACKAGES: {{ upload }}
+    PLATFORM: {{ platform }}
+    DOCKER_IMAGE: {{ docker.image }}
```

Comment:
It's not `docker.image` that should be used here. We should return the full config in `configs` and can query `docker_image` from that config. Different configs can have different docker images in other CI. We should have the same here